packages/node/src/protocol/install-virtual-app.ts

```diff
@@ -787,6 +793,53 @@ export const INSTALL_VIRTUAL_APP_PROTOCOL: ProtocolExecutionFlow = {
   }
 };
 
+// todo(xuanji): make this more consistent with the function
+// with the same name from install.ts. This involves refactoring
+// the callers.
+function computeInterpreterParameters(
```

Comment:
Write a docstring for this function please; or re-use the function in the other file which has:

https://github.com/counterfactual/monorepo/blob/837d5b29e05311b63e0fad0f2fe5ec6be73c5b59/packages/node/src/protocol/install.ts#L360-L382